apis/core/types_deployitem.go

```diff
@@ -54,6 +54,9 @@ type DeployItemSpec struct {
 	// Note that the type information is used to determine the secret key and the type of the secret.
 	// +optional
 	RegistryPullSecrets []ObjectReference `json:"registryPullSecrets,omitempty"`
+	// PickupTimeout is the time in seconds after which the DeployItem goes into a failure state if it has not been picked up by any deployer.
+	// +optional
+	PickupTimeout *int64 `json:"pickupTimeout,omitempty"`
```

Comment:
not sure if this timeout should be configured on the deploy item.
I think the pickup timeout should be configured in the landsacper config.

When the watlk about the overall deploy item timeout, this should be configurable per deploy item

Reply:
Hm, I didn't think of this, but it sounds reasonable. Will implement it this way.